### ISSUE TITLE
Remove Fonts as a peer dependency

### ISSUE THREAD

**Issue by jtrujill** [HealthCatalyst/Fabric.Cashmere#274]
Rather than have a peer dependency on OpenSans it would be better instead to have developers link to the company's standard font-family using a `<link rel="stylesheet" type="text/css" href="">` in their html head section. When we define the font-family for our components we should have fallback fonts. 

The main point is that it'd be more beneficial to download the font through Google CDN network and help reduce the size of our app payloads. Then as part of the style guide it would just be mentioned that we use "Open Sans" as a font as a company.

While open sans may not get updates frequently, which means we won't run into version issues between the libraries and user's font version, that's not the case with fontawesome. Fontawesome released version 5 which changes the names of some icons and is a whole new redesign. Now if we were using v5 but the user's app used v4, and assuming that no css names conflict, the user would be required to include fontawesome v4 and v5 or be required to do a full conversion of their app to avoid that.

Instead we should remove fontawesome references within the library and use css tricks as replacements. It seems all our references should be easy to remove except maybe the table css which uses the different sorting icons, but from a quick look it seems like we should be able to pull what we need out using something like http://fontello.com/.

**Comment by jtrujill** [comment]
I've looked up whether their are conflicts with using FontAwesome 4 and 5 and there are issues when both fontSets are loaded in parallel. By having FontAwesome as a dependency of this library we are effectively blocking all users from upgrading to the latest. I'm going to start removing all references to FontAwesome from Cashmere.

**Comment by jtrujill** [comment]
closing as a duplicate of #282
